site/en/docs/web-platform/conditional-focus/index.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Better screen sharing with Conditional Focus'
+description: >
+ Conditionally focus a tab or window when screen sharing on the web.
+authors:
+  - beaufortfrancois
+  - eladalon
+date: 2022-11-16
+hero: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/O12v673aOekK889d5Upi.jpg
+alt: Person holding eyeglasses with distant items in focus through the glasses.
+tags:
+  - chrome-109
+---
+
+The [Screen Capture API] lets the user select a tab, window, or screen to capture as a media stream. This stream can then be recorded or shared with others over the network. This documentation introduces Conditional Focus, a mechanism for web apps to control whether the captured tab or window will be focused when capture starts, or whether the capturing page will remain focused.
+
+## Browser support {: #browser-support }
+
+Conditional Focus is available from Chrome&nbsp;109.
+
+## Background
+
+When a web app starts capturing a tab or a window, the browser faces a decision—should the captured surface be brought to the forefront, or should the capturing page remain focused? The answer depends on the reason for calling `getDisplayMedia()`, and on the surface the user ends up selecting.
+
+Consider a hypothetical video conferencing web app. By reading `track.getSettings().displaySurface` and potentially examining the [Capture Handle], the video conferencing web app can understand what the user chose to share. Then:
```

Comment:
I wonder if we should make "hypothetical" bold and make the two bold statements below non-bold. Wdyt?

Reply:
It makes sense to me. Note that it was part of our original doc.

Reply:
Noted.